Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,18 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static bool containsInvalidCharacters(const String& str)
+{
+    for (unsigned i = 0; i < str.length(); i++) {
```

Comment:
I think this can use `const auto& character : str`

Reply:
`auto&` without the `const` is more idiomatic in WebKit

Reply:
I changed it use `String::contains` as recommended below.